downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc

```diff
@@ -4,7 +4,7 @@
 = Deploying your system for container signing
 
 
-To deploy your system so that it is ready for container signing, first ensure that you have link:{URLContainerizedInstall}/aap-containerized-installation#enabling-automation-hub-collection-and-container-signing_aap-containerized-installation[enabled automation content collection and container signing]. Then you can create a signing script, or xref:proc-adding-an-execution-environment.adoc[add and sign an {ExecEnvShort}] manually.
+To deploy your system so that it is ready for container signing, first ensure that you have link:{URLContainerizedInstall}/aap-containerized-installation#enabling-automation-hub-collection-and-container-signing_aap-containerized-installation[enabled automation content collection and container signing]. Then you can create a signing script, or xref:proc-adding-an-execution-environment[add and sign an {ExecEnvShort}] manually.
```

Comment:
```suggestion
To deploy your system so that it is ready for container signing, first ensure that you have link:{URLContainerizedInstall}/aap-containerized-installation#enabling-automation-hub-collection-and-container-signing_aap-containerized-installation[enabled automation content collection and container signing]. Then you can create a signing script, or xref:adding-an-execution-environment[add and sign an {ExecEnvShort}] manually.
```

Reply:
The ID in the xref was causing a build failure: it should be xref:adding-an-execution-environment